components/patient-data-sharing/push-ui/src/main/resources/PhenoTips/PushPatient.xml

```diff
@@ -839,6 +859,55 @@
     },
 
     // ---------------------------------------------------------------------
+    // Generate consents section UI
+    // ---------------------------------------------------------------------
+    _populateConsentsList: function() {
+      var _this = this;
```

Comment:
also, a leading underscore is usually used to signify private properties, so this is kind of confusing.